### ISSUE TITLE
Can't remove transparent obsidiat from some ore dict entries

### ISSUE THREAD

**Issue by Bulldog83** [An-Sar/PrimalCore#261]
PrimalCore 0.6.90
Forge 14.23.4.2747

I try to remove transparent obsidian, all of it's variants, from ore dict entries blockGlass, blockGlassColorless and glassReinforced via CraftTweaker. But these don't removes.

**Comment by An-Sar** [comment]
hm this is strange. have you run into this problem with any other primal items?

**Comment by Bulldog83** [comment]
Yes, also I remove Iron Glass from same entries, and it's removed.

**Comment by An-Sar** [comment]
weird, they are getting their names the same way, the main difference i guess is obsidian has meta values..
